### PR TITLE
Test reduction surgery 2: Re-organized all of the already-small(?) test data into data/tests

### PR DESCRIPTION
**Pull request by hepcat72**
## Summary Change Description

Based on Lance's feedback on the issue, I was 3 commits into the next PR and decided to make each commit into a PR, even though they may not pass linting or tests. These changes will not be merged until the head of the effort passes.

## Affected Issues/Pull Requests

- Partially addresses #480

## Review Notes

See comments in-line.

## Checklist

This pull request will be merged once the following requirements are met.  The
author and/or reviewers should uncheck any unmet requirements:

- Review requirements
  - Minimum approvals: 1 
  - No changes requested
  - All blocking issues resolved by reviewers
  - Specific reviewers: @__add_username_here__
    
  - Review period: 2 days 
- Associated issue/pull request requirements:
  
  - [x] All requirements in affected issues marked "resolved" are satisfied
  - [x] All required pull requests are merged *(or none)*
- Basic requirements
  
  - [ ] [All linters pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#linting)
  - [ ] [All tests pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#quality-control)
  - [x] All conflicts resolved
- Overhead requirements
  
  - [x] [New/changed method tests implemented/updated *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
  - [x] Added changes to *Unreleased* section of [`changelog.md`](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/changelog.md)
  - [x] [Migrations created & committed *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)

### CODE REVIEW

File: DataRepo/data/tests/accucor_with_multiple_labels/accucor.xlsx
Comment (hepcat72) [inline DataRepo/data/tests/accucor_with_multiple_labels/accucor.xlsx:1]: C. All changes are related to simply re-arranging some example/test data. I moved and/or renamed a bunch of directories and files, but definitely not all files. Paths to those files were changed via a search and replace script in the code/files.

**Review by lparsons — APPROVED** [review]
Simple move of files, though I don't understand the overall branching strategy or what I should review here really.

**Comment by hepcat72** [comment]
> Simple move of files, though I don't understand the overall branching strategy or what I should review here really.

During my work on "test_reduction_surgery2", I was on my third commit when you suggesting breaking it up more, so I just:
- Branched the HEAD with `test_reduction_surgery3`
- Switched to `test_reduction_surgery2` and did a `git reset --hard HEAD^`
- Discovered I had another commit before this one and checked that out and made it `test_reduction_surgery1.5` (If I'd realized there were 3 commits, I'd have done 4, 3, and 2.)

**Comment by lparsons** [comment]
> During my work on "test_reduction_surgery2", I was on my third commit when you suggesting breaking it up more, so I just:

Sure, though my point was to break it up so that tests passed for each PR and each PR changed only one test class at a time to make it easier to understand when reviewing. In fact, the entire test class doesn't even have to be changed in one PR. But as long as we get there, it's fine.

**Comment by hepcat72** [comment]
> > During my work on "test_reduction_surgery2", I was on my third commit when you suggesting breaking it up more, so I just:
> 
> Sure, though my point was to break it up so that tests passed for each PR and each PR changed only one test class at a time to make it easier to understand when reviewing. In fact, the entire test class doesn't even have to be changed in one PR. But as long as we get there, it's fine.

Just was trying to explain the branching since you said you didn't understand it. And yes, I get your suggestion. Aside from the re-org, I'm doing it by dataset, which makes more sense to me. And of course, now that I understand a few datasets at a time is too much, I can definitely ensure they pass.  This was just a retroactive attempt at somewhat accommodating your request.

**Pull request merged**